### PR TITLE
458 - reduced spacing between features and FAQ on landing page

### PR DESCRIPTION
**Pull request by kellysu1**
CHANGES: 
fixed the spacing for mobile and tablet sized view for the landing page, in between features and FAQs. 
Added tests for features to make sure everything is still visible. 

HOW TO VIEW: 
Go to landing page and scroll until the end of features and start of FAQs 

Attached is a screenshot of the new spacing. 

Screen Shot 2024-03-28 at 6 43 52 pm

### CODE REVIEW

**Review by richard-xue-vrx — APPROVED** [review]
Looks perfect to me; spacing is fixed on tablet and mobile view w/ local inspection. Nice job on the tests too!

File: frontend/__tests__/Features.test.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    const heading = screen.getByText("Our Features");
+
+    expect(heading).toBeInTheDocument();
+  });
+
+  it("shows BrowseBuildings icon", () => {
+    render(<Features />);
+
+    const BrowseBuildings = screen.getByText("Browse Buildings");
```
Comment (JessicaF) [inline frontend/__tests__/Features.test.tsx:24]: Small nit but can we use camelCase for variables in tests (same for the below tests)

**Review by JessicaF — CHANGES_REQUESTED** [review]

**Comment by JessicaF** [comment]
I'm having issues on my local site where the Our Features heading is covered by the Browse Buildings block when on mobile/tablet sized screens.

Screenshot 2024-04-04 at 10 41 21 PM

Also, the boxes end up touching on some screen sizes - could you make it so that there's always a gap on all screen sizes?
Screenshot 2024-04-04 at 10 43 46 PM

**Comment by kellysu1** [comment]
Hi have addressed all the requests 
- Our features is not covered anymore when in mobile and tablet view 
Screen Shot 2024-04-17 at 4 46 18 pm

- There is now spaces between each box for the features 
Screen Shot 2024-04-17 at 4 46 07 pm

- the naming is also fixed in the testing

**Comment by JessicaF** [comment]
Thanks for the fix!!

**Pull request merged**